turbopy/core.py

```diff
@@ -300,9 +300,10 @@ def _registry(self):
         pass
 
     @classmethod
-    def register(cls, name_to_register: str, class_to_register):
+    def register(cls, name_to_register: str, class_to_register,
+                 override=False):
         """Add a derived class to the registry"""
-        if name_to_register in cls._registry:
+        if name_to_register in cls._registry and not override:
```

Comment:
Aren't these changes already in `main`? Did you merge any changes from main back into your dev branch before doing a PR?

Reply:
These changes got nuked somehow. IDK why.

Reply:
I just checked and these changes are indeed in my version that on my local machine.  IDK why they are not present in my pull request.